diff --git a/node_modules/react-native-localize/RNLocalize.podspec b/node_modules/react-native-localize/RNLocalize.podspec
index 4a161ee..3f66d43 100644
--- a/node_modules/react-native-localize/RNLocalize.podspec
+++ b/node_modules/react-native-localize/RNLocalize.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author       = package["author"]
   s.homepage     = package["homepage"]
 
-  s.platforms    = { :ios => "12.4", :tvos => "12.4", :osx => "10.15" }
+  s.platforms    = { :ios => "12.4", :tvos => "12.4", :osx => "10.15", :visionos => "1.0" }
   s.requires_arc = true
 
   s.source       = { :git => package["repository"]["url"], :tag => s.version }
